src/scss/components/_methodology.scss

```diff
@@ -0,0 +1,57 @@
+@use 'abstracts' as *;
+
+.methodology {
+  position: relative;
+  display: grid;
+  grid-column-gap: 72px;
+  grid-template-columns: 2fr 1fr;
+  padding-top: rem(60px);
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
+    display: block;
+    width: 100vw;
+    height: 1px;
+    background-color: #979797;
+  }
+
+
+  &__title {
+    grid-column: 1 / span 2;
+    margin-top: 0;
+    color: $color__white;
+    font-family: $font__expo;
+    font-weight: 600;
+    font-style: normal;
+    @include font-size(34px);
+    font-stretch: normal;
+    line-height: 1;
+    letter-spacing: normal;
+  }
+  
+  &__desc {
+    grid-column: 1 / span 1;
+    grid-row: 2 / span 1;
+    color: $color__white-190;
+    @extend %body__text-l;
+  }
+  
+  &__program-desc {
+    padding-left: rem(24px);
+    color: $color__white-190;
+    @extend %body__text-m;
+
+    a {
+      color: $color__blue-200;
+      transition: all 0.5s ease;
+
+      &:hover {
+        color: #89b9dc;
```

Comment:
Let's add this blue to the variables file, as we'll use it in the footer as well for the links.